Layouts/default.ctp

```diff
@@ -10,6 +10,16 @@
     <?= $this->Html->css('new-style.css') ?>
     <?= $this->Html->css('https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css') ?>
     <?= $this->Html->script('jquery-2.2.4.js') ?>
+    <meta property="og:title" content="<?= $website_name ?>">
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="<?php echo "http://".$_SERVER['SERVER_NAME'].$_SERVER["REQUEST_URI"]; ?>">
```

Comment:
`<?php echo "XXX"; ?>` is the same as `<?= "XXX" ?>`, so it's better to use this one